libs/books/data-access/src/lib/+state/reading-list.reducer.ts

```diff
@@ -47,11 +47,17 @@ const readingListReducer = createReducer(
       error: action.error
     };
   }),
-  on(ReadingListActions.addToReadingList, (state, action) =>
-    readingListAdapter.addOne({ bookId: action.book.id, ...action.book }, state)
+  on(ReadingListActions.addToReadingList, (state, action) => {
+    return readingListAdapter.addOne({ bookId: action.book.id, ...action.book }, state)}
```

Comment:
Don't change the convention half way through the file, there's no need for these bracketed return statements.

Reply:
reverted